plasmapy/particles/particle_class.py

```diff
@@ -409,6 +380,70 @@ def __init__(
 
             categories.add(Element['category'])
 
+
+
+
+
+##########################
+
```

Comment:
Delete lines...[PEP8: there should be a single blank line between methods](https://www.python.org/dev/peps/pep-0008/#blank-lines)